packages/ListBox/stories/examples/Keyboard/index.js

```diff
@@ -0,0 +1,337 @@
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              <ListBox.Option isDisabled>Achilles</ListBox.Option>
+              <ListBox.Option isDisabled>Odysseus</ListBox.Option>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            The user is allowed to navigate over disabled elements but can not selected them or interact with them. This
+            might required further discussion of what we really want to do.
+          </CheckListItem>
+        </Block>
+      </Blocks>
+    </>
+  );
+}
+
+export function WithRawItems() {
+  return (
+    <>
+      <h2>With Option.RawItem</h2>
+      <Blocks>
+        <Block>
+          <h3>Single</h3>
+
+          <Card>
+            <ListBox>
+              <ListBox.Filter />
+              <ListBox.Divider>Raw-Items</ListBox.Divider>
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Achilles
+              </ListBox.RawItem>
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Odysseus
+              </ListBox.RawItem>
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+          <h3>Multi</h3>
+
+          <Card>
+            <ListBox isMulti>
+              <ListBox.Filter />
+              <ListBox.Divider>Raw-Items</ListBox.Divider>
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Achilles
+              </ListBox.RawItem>
+              <ListBox.RawItem
+                onClick={() => {
+                  console.log("RawItem clicked");
+                }}
+              >
+                Odysseus
+              </ListBox.RawItem>
+              <ListBox.Divider>Anti-heroes</ListBox.Divider>
+              {characters.antiHeroes}
+              <ListBox.Divider>Heroes</ListBox.Divider>
+              {characters.heroes}
+              <ListBox.Divider>Villians</ListBox.Divider>
+              {characters.villians}
+            </ListBox>
+          </Card>
+        </Block>
+        <Block>
+          <CheckListItem>
+            The user is allowed to navigate over disabled elements but can not selected them or interact with them. This
+            might required further discussion of what we really want to do.
```

Comment:
Can we apply the same "subtle" focus ring to raw items that we do for disabled items?

```js
${stylers.focusRing.subtle()}
```